src/RAJAPerfSuiteDriver.cpp

```diff
@@ -10,26 +10,44 @@
 
 #include <iostream>
 
+#ifdef RAJA_PERFSUITE_ENABLE_MPI
+#include <mpi.h>
+#endif
+
 //------------------------------------------------------------------------------
 int main( int argc, char** argv )
 {
+#ifdef RAJA_PERFSUITE_ENABLE_MPI
+  MPI_Init(&argc, &argv);
+#endif
+
+#ifdef RAJA_PERFSUITE_ENABLE_MPI
```

Comment:
why is this in a separate macros guarded section? move to section above with mpi init